playground/ci/find_beam_examples.py

```diff
@@ -0,0 +1,76 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import re
+import logging
+from typing import List
+
+EXTENSIONS = ["java", "go", "py"]
+PATTERN = re.compile('Beam-playground:\n {4}name: \w+\n {4}description: .+\n {4}multifile: (true|false)\n {4}categories:\n( {8}- [\w\-]+\n)+')
+
+
+def get_examples():
+    """Get all beam examples
+
+    Find and return file path for each beam example
+
+    Returns:
+        List of examples' file path.
+        If root dir is not present in environment returns None.
+    """
+    root_dir = os.getenv("BEAM_ROOT_DIR")
```

Comment:
It is a bad idea to get environment variables in such functions, here you should take it as an argument of the function.

Reply:
Changed.